.eslintrc.json

```diff
@@ -21,6 +21,7 @@
     "node": true
   },
   "rules": {
+    "jsx-a11y/label-has-associated-control": 0,
```

Comment:
adding this rule here affects the whole project. It should be possible to add an ignore rule only in the interested file, which is a lot better.